addons/point_of_sale/tests/common.py

```diff
@@ -62,3 +63,9 @@ def setUp(self):
         # I assign those 5 percent taxes on the PCSC349 product as a sale taxes
         self.product4.write(
             {'taxes_id': [(6, 0, [account_tax_05_incl.id, account_tax_05_incl_chicago.id])]})
+
+        invoice_rep_lines = (account_tax_05_incl | account_tax_05_incl_chicago | account_tax_10_incl).mapped('invoice_repartition_line_ids')
+        refund_rep_lines = (account_tax_05_incl | account_tax_05_incl_chicago | account_tax_10_incl).mapped('refund_repartition_line_ids')
+
+        tax_received_account = self.env['account.account'].search([('name', '=', _('Tax Received'))])
+        (invoice_rep_lines | refund_rep_lines).write({'account_id': tax_received_account.id})
```

Comment:
missing empty line at the end of file (when a line is added, the small red symbol disapear but you won't see the line on github)